ui/compontents.go

```diff
@@ -803,6 +803,151 @@ func (page pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount,
 	}
 }
 
+func ticketLiveItemnInfo(gtx layout.Context, c pageCommon, t *wallet.Ticket) layout.Dimensions {
+	var itemWidth int
+	st := ticketIconStatus(&c, t.Info.Status)
+	if st == nil {
+		return layout.Dimensions{}
+	}
+	st.icon.Scale = 1.0
+	return c.theme.Shadow(c.theme.Color.LightGray).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				wrap := c.theme.Card()
+				wrap.Radius.NE = 8 // top - left
+				wrap.Radius.SW = 0 // bottom - left
+				wrap.Radius.NW = 8 // top - right
+				wrap.Radius.SE = 0 // bottom - right
```

Comment:
* each corner radius doesn't have to be set on a new line. You can define the radius using the * * * 
CornerRadius type on a single line and assign it once. As in 
```
radius := decredmaterial.CornerRadius{8, 0, 8, 0}
wrap.Radius = radius
```
* I believe NE (north east) is top right not top left. 
* I'm not sure "ticketLiveItemnInfo" is a suitable name for this. "ticketCard" maybe?
* Can comments be added to this layout? So that it is easier to read and understood quicker.